boards/nxp/frdm_mcxc444/frdm_mcxc444.dts

```diff
@@ -36,7 +36,7 @@
 			label = "Red LED";
 		};
 		green_led: led_1 {
-			gpios = <&gpiob 5 GPIO_ACTIVE_LOW>;
+			gpios = <&gpiod 5 GPIO_ACTIVE_LOW>;
```

Comment:
It looks more like you are "fixing the DT LED definitions" than "Update led pin align with board SCH" :) You should just say it in the commit title and message.